test/test_linalg.py

```diff
@@ -4746,15 +4743,20 @@ def test_bmm(self, device, dtype):
         M, N, O = 23, 8, 12
         numpy_dtype = dtype if dtype != torch.bfloat16 else torch.float32
 
-        if self.device_type == 'cpu':
-            is_supported = True
-        elif self.device_type == 'cuda':
-            is_supported = True if dtype != torch.bfloat16 else AMPERE_OR_ROCM
+        is_supported = True
+        is_cuda_bfloat = False
+        if dtype == torch.bfloat16 and self.device_type == 'cuda':
+            is_supported = TEST_WITH_ROCM or (CUDA11 and SM53)
+            is_cuda_bfloat = True
 
         if not is_supported:
-            b1 = torch.randn(num_batches, M, N, device=device).to(dtype)
-            b2 = torch.randn(num_batches, N, O, device=device).to(dtype)
-            self.assertRaisesRegex(RuntimeError, "type|Type|not implemented|Ampere", lambda: torch.bmm(b1, b2))
+            if not is_cuda_bfloat:
```

Comment:
`is_supported=False`, `is_cuda_bfloat=False` is an impossible situation?

Reply:
Some ops are supported on SM52, and some are not. I don't think it worth the maintenance effort to get a clear list on which is supported on which SM. So what I implemented here is:
```
SM >= 53 ---> supported
SM < 53 ---> undefined behavior
```